datalake/src/main/resources/duties/7.2.3/yarn/medium_duty_ha.json

```diff
@@ -27,9 +27,20 @@
           "memory": 16384
         }
       },
-      "nodeCount": 1,
+      "nodeCount": 2,
       "type": "GATEWAY"
     },
+    {
+      "name": "alpha",
```

Comment:
Could we give it a more meaningful name than "alpha"?

Reply:
changed to auxiliary and master to main.